althea_kernel_interface/src/lib.rs

```diff
@@ -46,6 +47,9 @@ impl KernelInterface {
     fn run_command(&mut self, program: &str, args: &[&str]) -> Result<Output, Error> {
         let output = Command::new(program).args(args).output()?;
         trace!("Command {} {:?} returned: {:?}", program, args, output);
+        if !output.status.success() {
+            trace!("An error was returned");
```

Comment:
Why?

Reply:
I was grepping to see what commands failed, and there were too many different error codes that can occur lol